src/views/ConsentContinuationOut.vue

```diff
@@ -612,7 +612,7 @@ export default class ConsentContinuationOut extends Vue {
     const data: any = {
       [FilingTypes.CONSENT_CONTINUATION_OUT]: {
         // FUTURE: add more filing properties below
-        comment: `${this.defaultComment}\n${this.detailComment}`
+        details: `${this.defaultComment}\n${this.detailComment}`
```

Comment:
Two bugfixes in here...

According to [the schema](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/consent_continuation_out.json), the property is called details (not comment) and the max length is 2000.